app/controllers/tutorials_controller.rb

```diff
@@ -1,9 +1,6 @@
 class TutorialsController < ApplicationController
-  def index
-    @tutorials = Tutorial.includes(:tutorial_steps).all
-  end
-
   def show
-    @tutorial = Tutorial.includes(:tutorial_steps).find(params[:id])
+    @tutorials = Tutorial.includes(:tutorial_steps).all
```

Comment:
It looks like `@tutorials` is used to populate the side navigation with links to other `tutorials#show` views. If that's the case, it's probably not necessary to `.includes(:tutorial_steps)`.

```rb
@tutorials = Tutorial.all
```

☝️ That should suffice (unless I've missed something).

Reply:
Oh… never mind. I see what line 4 is doing…